modules/images/auto-sizes/load.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Module Name: Auto Sizes for Lazy-loaded Images.
+ * Module Name: Auto Sizes for Lazy-loaded Images
```

Comment:
It's not a blocker, but in the [CODEOWNERS file](https://github.com/WordPress/performance/blob/feature/auto-sizes/.github/CODEOWNERS#L57), we have `Auto-sizes for Lazy-Loaded Images` specified, and the module name differs from this designation. Should we consider correcting this discrepancy for consistency?

Reply:
Good observation! Looking at the official spec and various references from early browser implementers, I see this both hyphenated and not, but it's good for us to be consistent, so I've updated all references to be hyphenated in [6870baf](https://github.com/WordPress/performance/pull/922/commits/6870baf52050e5e1240db347cd7d55670b4d58bb).